cmd/main.go

```diff
@@ -113,8 +112,6 @@ func InitFlags(fs *pflag.FlagSet) {
 }
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
interesting, what I [did](https://github.com/kubernetes-sigs/cluster-api-operator/pull/187/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R116-R117) was a bit different than this..